api/src/main/kotlin/xyz/bukinator/api/application/dto/response/HousesResponse.kt

```diff
@@ -0,0 +1,29 @@
+package xyz.bukinator.api.application.dto.response
+
+import xyz.bukinator.api.domain.house.House
+
+data class HousesResponse(
+    val data: List<HouseSummaryResponse>
+) {
+    companion object
+}
+
+data class HouseSummaryResponse(
+    val id: Long,
+    val salesType: String,
+    val thumbnail: String,
+    val priceDeposit: Int,
+    val priceRent: Int,
+    val title: String,
+    val addressJibun: String
+) {
+    constructor(house: House) : this(
```

Comment:
나는 constructor 를 더 좋아하는 편이지만 named constructor 가 권장되기는 해서 제안)

```suggestion
    companion object {
        fun from(house: House): HouseSummaryResponse {
            ...
        }
    }
```

Reply:
요것도 컨벤션 잡아보자.

나도 코드 찾아봤을대는 named constructor를 사용하는 경우가 더 많았던 것 같았습니다

Reply:
난 그냥 constructor 쓰는것도 조음